.golangci.yml

```diff
@@ -31,11 +31,16 @@ linters-settings:
       - name: range
       - name: receiver-naming
       - name: time-naming
-      - name: unexported-return
```

Comment:
What do you think to capturing the reasons for disabling linters? Perhaps as a ADR or doc somewhere in the repo.

Reply:
actually, we can use this linter. I removed it because it didn't allow to return private structures in public functions, now I replaced structure with the interface.

Reply:
Nice